app/controllers/agent/nextPaymentDue/PaymentDueController.scala

```diff
@@ -62,7 +62,7 @@ class PaymentDueController @Inject()(paymentDue: paymentDue,
     paymentDue.apply(
       chargesList = charge,
       currentTaxYear = taxYear,
-      paymentEnabled = isEnabled(Payment),
```

Comment:
Please remove this from the view instead of setting it to true

Reply:
done